### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JonathanTurnock**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+DCS_DZ_TYPEORM_LOGGER='simple-console'
```
Comment (john681611) [inline .env:1]: [question] is this supposed to be commited?

**Review by john681611 — COMMENTED** [review]
There is simply too many files and changes to review in detail so I got manual testing

Regestry URL is borked:
image


Bit of this too: 
```
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:17:53     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:00     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:00     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:00     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:01     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:01     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:01     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:01     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:02     LOG [SubscriptionManager] Subscribing to mod example-mod
[Nest] 11268  - 24/07/2024, 21:18:02     LOG [SettingsManager] Using registry URL: undefined
[Nest] 11268  - 24/07/2024, 21:18:02     LOG [SettingsManager] Using registry URL: undefined
Skip checkForUpdates because application is not packed and dev update config is not forced
checkForUpdatesAndNotify called, downloadPromise is null
Skip checkForUpdates because application is not packed and dev update config is not forced
checkForUpdatesAndNotify called, downloadPromise is null
Skip checkForUpdates because application is not packed and dev update config is not forced
checkForUpdatesAndNotify called, downloadPromise is null
Skip checkForUpdates because application is not packed and dev update config is not forced
checkForUpdatesAndNotify called, downloadPromise is null
Skip checkForUpdates because application is not packed and dev update config is not forced
checkForUpdatesAndNotify called, downloadPromise is null
```

File: .env (hunk 1 above)
Comment (JonathanTurnock) [reply .env]: Just binned that :)

**Comment by JonathanTurnock** [comment]
Removed the SettingsManager line, that was just because something was not behaving
Also fixed the RegistryURL

**Pull request merged**